next/components/styleguide/showcases/SelectFieldShowCase.tsx

```diff
@@ -28,7 +33,12 @@ const SelectFieldShowCase = () => {
         <SelectField name="select-test" label="Select Field" options={selectOptions} disabled/>
       </Stack>
       <Stack>
-        <SelectField name="select-test" label="Select Field" options={selectOptions} required/>
+        <SelectField name="select-test" label="Select Field" options={selectOptions} required ref={ref}
+                     value={selectValueSecond}
+                     onChange={value => {
+                       setSelectValueSecond(value)
+                       console.log(ref.current?.getValue())
```

Comment:
was here likely just for debug and can be removed

Reply:
Yes this is on purpose, just for showing that ref is working.